[WebAuthn] Update forward declarations according to internal needs

pascoej opened the pull request:
#### 6d162ad395016d14a07b6f78caebfb6526e18a17
<pre>
[WebAuthn] Update forward declarations according to internal needs
<a href="https://bugs.webkit.org/show_bug.cgi?id=241992">https://bugs.webkit.org/show_bug.cgi?id=241992</a>
rdar://problem/95887698

Reviewed by Brent Fulgham.

* Source/WebKit/Platform/spi/Cocoa/AuthenticationServicesCoreSPI.h:
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/AuthenticationServicesCoreSoftLink.h:
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/AuthenticationServicesCoreSoftLink.mm:
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm:
(WebKit::setGlobalFrameIDForContext):

Canonical link: <a href="https://commits.webkit.org/251843@main">https://commits.webkit.org/251843@main</a>
</pre>

brentfulgham reviewed (approved):
When did the rename of ASCGlobalFrameIdentifier -> ASGlobalFrameIdentifier happen?

Do we need to do some kind of versioning to avoid breaking the build until everyone is on an SDK with the new symbol?

pascoej committed:
[WebAuthn] Update forward declarations according to internal needs
https://bugs.webkit.org/show_bug.cgi?id=241992
rdar://problem/95887698

Reviewed by Brent Fulgham.

* Source/WebKit/Platform/spi/Cocoa/AuthenticationServicesCoreSPI.h:
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/AuthenticationServicesCoreSoftLink.h:
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/AuthenticationServicesCoreSoftLink.mm:
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm:
(WebKit::setGlobalFrameIDForContext):

Canonical link: https://commits.webkit.org/251843@main
